_includes/page_description.html

```diff
@@ -0,0 +1,3 @@
+<div markdown="1">
+  <h4 class='text-center'>{{ include.description }}</h4>
```

Comment:
Did you mean to make this `<h4>`? You may want to rename these to make it clearer where page_description is used.